src/k8sconfiguration/README.rst

```diff
@@ -0,0 +1,5 @@
+Microsoft Azure CLI 'k8sconfiguration' Extension
+==========================================
+
+This package is for the 'k8sconfiguration' extension.
+i.e. 'az k8sconfiguration'
```

Comment:
Please add content for README file. You can reference [here](https://github.com/Azure/azure-cli-extensions/blob/master/src/stream-analytics/README.md).

Reply:
Added